lms/static/scripts/ui-playground/index.ts

```diff
@@ -0,0 +1,12 @@
+// Entry point for local webserver pattern-library bundle
+import { startApp } from '@hypothesis/frontend-shared/lib/pattern-library';
+import type { CustomPlaygroundRoute } from '@hypothesis/frontend-shared/lib/pattern-library/routes';
+
+// LMS prototype pages should be defined here
+const extraRoutes: CustomPlaygroundRoute[] = [];
```

Comment:
This could be removed now, but I left it for future reference of where are pages supposed to be configured.